src/components/Message.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default class Message extends React.PureComponent {
+  render() {
+    return (
+      <>
```

Comment:
do we need this redundant fragment?

Reply:
For example, I transferred the style to the App.css style file and imported the style. As I understand it, if a style is set as an object and used in a component, it is not visible to other components, it doesn't allow reuse. Thanks for your note.

Reply:
Yes, not needed, I tried using Fragment in the example. <React.Fragment> or <> can create a list of child elements, without creating unnecessary nodes in the DOM. Thanks for your note.